.travis.yml

```diff
@@ -1,14 +1,17 @@
-sudo: false
-language: java
-dist: trusty
-jdk: oraclejdk8
-cache:
-  directories:
-  - "$HOME/.m2"
-install: true
-script:
-- mvn clean verify
-- sed -i'' -e "s/\$VERSION/${TRAVIS_TAG:=0.0.0}/" descriptor.json
-- sed -i'' -e "s/\$DATE/$(date +%Y-%m-%d)/" descriptor.json
-notifications:
-  webhooks: https://fathomless-fjord-24024.herokuapp.com/notify
+dist: bionic
+
+language: java 
+jdk:
+  - openjdk8
+  - openjdk11
+
+#-----start GUI testing stuff
+services:
+  - xvfb
+#-----end GUI testin stuff

```

Comment:
fix typo and remove "stuff" please

Reply:
This has already been fixed. See: https://github.com/tcerny/VariantSync/tree/issue-52